website/src/documentation/dsls/sql/calcite/overview.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: section
+title: "Beam SQL in Calcite: Overview"
+section_menu: section-menu/sdks.html
+permalink: /documentation/dsls/sql/calcite/overview/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Beam SQL Overview (Calcite)
```

Comment:
perhaps this, to be consistent with the overall overview page title?

Beam SQL: Calcite support overview